examples/counter/src/lib.rs

```diff
@@ -3,10 +3,9 @@
 
 use async_graphql::{Request, Response};
 use linera_sdk::base::{ContractAbi, ServiceAbi};
-use serde::{Deserialize, Serialize};
+use serde::Deserialize;
 
-// TODO(#768): Remove the derive macros.
-#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+#[derive(Deserialize)]
```

Comment:
We also want to remove `Deserialize` !

Reply:
These ones don't seem to compile if I remove them. I'll look into the error more carefully and try to solve it

Reply:
It's because of the buggy where clause in `identifiers.rs`
```
 impl<'de, A> Deserialize<'de> for ApplicationId<A>
where
    A: Deserialize<'de>,
```
Let me fix this really quick

Reply:
See #864